Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -1994,4 +1999,23 @@ void WebChromeClient::didProgrammaticallyClearTextFormControl(const HTMLTextForm
     protectedPage()->didProgrammaticallyClearTextFormControl(element);
 }
 
+#if ENABLE(DAMAGE_TRACKING)
+void WebChromeClient::resetDamageHistoryForTesting()
+{
+    const auto* layerTreeHost = static_cast<LayerTreeHost*>(graphicsLayerFactory());
```

Comment:
LayerTreeHost can be nullptr in gtk if accelerated compositing is disabled. I don't think we should be using LayerTreeHost from here, I would use the drawing area that is always available and that can access its layer tree host if present.

Reply:
thanks, I've done this through DrawingArea, as suggested, so we don't enable AC if layerTreeHost is not yet instantiated